src/haz3lweb/view/TestView.re

```diff
@@ -23,23 +23,27 @@ let test_instance_view =
     ],
   );
 
-let jump_to_test = (~inject as _, _) => Effect.Ignore;
+let jump_to_test = (~inject, pos, id, _) => {
+  let effect1 = inject(Update.SwitchEditor(pos));
+  let effect2 = inject(Update.PerformAction(Jump(TileId(id))));
+  Effect.bind(effect1, ~f=_result1 => effect2);
```

Comment:
I think there is something like `Effect.many` you can use here instead

Reply:
Seems there isn't Effect.many in ReasonML?
Found Effect.all which "Combines the given effects into a single effect that runs them in parallel." Can't be applied here since the return type is not compatible.